mypy/options.py

```diff
@@ -191,7 +191,7 @@ def __init__(self) -> None:
         self.use_builtins_fixtures = False
 
         # -- experimental options --
-        self.shadow_file = None  # type: Optional[List[Tuple[str, str]]]
+        self.shadow_file = None  # type: Optional[List[List[str]]]
```

Comment:
Hm, I am curious why mypy didn't catch any error about this?

Reply:
The field gets populated by argparse, I think?